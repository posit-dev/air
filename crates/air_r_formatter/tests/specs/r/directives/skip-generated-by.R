# Generated by foo; do not edit

1 + 1



# bar



2



#foo


