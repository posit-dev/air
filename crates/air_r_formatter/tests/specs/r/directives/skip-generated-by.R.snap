---
source: crates/air_formatter_test/src/snapshot_builder.rs
info: r/directives/skip-generated-by.R
---
# Input

```R
# Generated by foo; do not edit

1 + 1



# bar



2



#foo



```


=============================

# Outputs

## Output 1

-----
Indent style: Space
Indent width: 2
Line ending: LF
Line width: 80
Persistent line breaks: Respect
-----

```R
# Generated by foo; do not edit

1 + 1



# bar



2



#foo


```
